File: Soft Tech Exam/Skeletons/CSharp/SoftUniAirlines/SoftUniAirlines/Models/Flight.cs
namespace SoftUniAirlines.Models
{
    using System.ComponentModel.DataAnnotations;


    public class Flight
    {
        public int Id { get; set; }

        [Required]
        public string Departure  { get; set; }

        [Required]
        public string Destination { get; set; }

        [Required]
        public string Status { get; set; }
    }
}

File: Soft Tech Exam/Skeletons/JavaScript/softuniairlines/models/flight.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    const Flight = sequelize.define("Flight", {
        departure: {type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },
        destination: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },
        status: {
            type: Sequelize.STRING,
            required: true,
            allowNull: false
        }
    }, {
        timestamps: false
    });

    return Flight;
};